runbot_build_instructions/runbot_repo_view.xml

```diff
@@ -15,6 +15,7 @@
             <field name="skip_test_jobs"/>
             <field name="custom_build_dir" attrs="{'invisible': [('is_custom_build', '=', False)]}"/>
             <field name="custom_server_path" attrs="{'invisible': [('is_custom_build', '=', False)], 'required': [('is_custom_build', '=', True)]}"/>
+            <field name="custom_odoo_path" attrs="{'invisible': [('is_custom_build', '=', False)]}"/>
```

Comment:
adding placeholder="odoo" maybe helpful if we have a implicit default value
But would block this